### ISSUE TITLE
list of special characters on password generator

### ISSUE THREAD

**Issue by acaetano** [PhilippC/keepass2android#1071]

**Comment by OldSunGuy** [comment]
This sounds like a great idea. I usually have to tap the generate password several times or edit one that's been created.

**Comment by Gerra-Diaphotos** [comment]
I agree. Some sites only allow a certain selection of special characters. Like in the desktop version of KeePass, it would be nice to be able to type them in a separate field for each entry individually.
Furthermore it should be possible to mask the password field of the generator, so that a screen reader cannot read it. Thank you!

![PWgen](https://user-images.githubusercontent.com/83730474/117208052-f8a32900-adf4-11eb-902b-2cab1f1bd566.PNG)

**Comment by acaetano** [comment]
Bump! as I still think this feature would not only be very useful, but also attract more users to Keepass2Android.
